lib/manifest.js

```diff
@@ -290,6 +290,20 @@ internals.manifest = [
         list: true,
         useFilename: internals.passthruOn('name')
     },
+    {
+        place: 'event/definitions',
+        method: 'event',
+        list: true,
+        useFilename: internals.passthruOn('name')
```

Comment:
P.S. this one for `event/definitions` is good, but needs to handle the case that an array is exported.  See the `routes` one as an example.